### PR TITLE
10.0 mig account bank statement import save file

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

File: account_bank_statement_import_save_file/__manifest__.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline account_bank_statement_import_save_file/__manifest__.py:1]: Can you update the README file and change the headers to the short ones?

**Review by elicoidal — CHANGES_REQUESTED** [review]

**Comment by elicoidal** [comment]
Have a look at travis as well

File: account_bank_statement_import_save_file/__manifest__.py (hunk 1 above)
Comment (mourad-ehm) [reply account_bank_statement_import_save_file/__manifest__.py:1]: Thanks @elicoidal, I maked the change. Could you have a look ?

**Review by elicoidal — APPROVED** [review]
LGTM altough increasing the code coverage wouldnot hurt.
I will not block for that though

File: account_bank_statement_import_save_file/i18n/fr.po [hunk 2]
```diff
@@ -30,9 +41,14 @@ msgstr "Importer Relevé Bancaire"
 #. module: account_bank_statement_import_save_file
 #: field:account.bank.statement,import_file:0
 msgid "Import file"
-msgstr ""
+msgstr "Importation de fichier"
```
Comment (alexis-via) [inline account_bank_statement_import_save_file/i18n/fr.po]: Import de fichier

**Review by alexis-via — APPROVED** [review]

**Comment by bealdav** [comment]
:+1:

**Pull request merged**